src/components/Footer.js

```diff
@@ -23,46 +23,34 @@ function Social_App({logo_img, url_link}) {
 
 export default function Footer() {
     /* mql = media query list (https://dev.to/yanns1/how-to-render-different-components-based-on-screen-size-2p35) */
-    const mql = window.matchMedia('(max-width: 600px)');
+    const mql = window.matchMedia('(max-width: 425px)');
```

Comment:
I'm not able to run this locally due to `node-sass` not supporting M1. I'm wondering if `window.matchMedia` will update on resizing the window? It's generally good practice to update the UI as the user resizes the window

Reply:
Yes the purpose of window.matchMedia is to adjust for different screen sizes. I tested on the local host for different screen sizes. I don't have an M1 chip so not sure if this would be an issue for all m chip users?

Reply:
Okay sweet! I get this error when I try to run the install.. I trust you here so I'll approve

```
npm ERR! gyp ERR! UNCAUGHT EXCEPTION 
npm ERR! gyp ERR! stack TypeError: Cannot assign to read only property 'cflags' of object '#<Object>'
npm ERR! gyp ERR! stack     at createConfigFile (/Users/rbernardo/Documents/GitHub/Advertisement-Website/node_modules/node-gyp/lib/configure.js:117:21)
npm ERR! gyp ERR! stack     at /Users/rbernardo/Documents/GitHub/Advertisement-Website/node_modules/node-gyp/lib/configure.js:84:9
npm ERR! gyp ERR! stack     at FSReqCallback.oncomplete (node:fs:180:23)
npm ERR! gyp ERR! System Darwin 22.5.0
npm ERR! gyp ERR! command "/opt/homebrew/Cellar/node/19.5.0/bin/node" "/Users/rbernardo/Documents/GitHub/Advertisement-Website/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd /Users/rbernardo/Documents/GitHub/Advertisement-Website/node_modules/node-sass
npm ERR! gyp ERR! node -v v19.5.0
npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! gyp ERR! Node-gyp failed to build your package.
npm ERR! gyp ERR! Try to update npm and/or node-gyp and if it does not help file an issue with the package author.
npm ERR! Build failed with error code: 7
```